### DISCUSSION TITLE
Detecting my robotic lawn mower as a person?

### DISCUSSION THREAD

**Discussion by mlg9000** [blakeblackshear/frigate#1420] [Ask A Question]
Frigate seems to detect my automower (robotic lawn mower) as a person regularly.  The degree of confidence it's saying that is almost the same as it does with an actual person (77% last event). It was also detecting it as a car too, which makes much more sense.  I was able to easily solve that issue by applying a mask for the car object over the lawn where a mower will go but an actual car won't.  I can't do the same for person detection.  

This is problematic as I use person detection to turn on outside lights and for my security setup.  The mower needs to do it's job will regularly pass in front of the cameras. If I raise the confidence level to 80%+ it seems like I will stop the issue but I will also miss a ton of actual person detection events.  So far I haven't collected enough events to be sure but I suspect this is a bigger issue at night with the night vision on the cameras (even though that is very good).

Anyone have ideas how to correct this?

**Comment by Lallassu** [comment]
I have the exact same issue. It's probably the trained model for the detection that requires some buff?
But it would be nice if there is a way to tackle this w/o a new model. My robot triggers the person detection like 10-20 times a day now, with a person detection rate around 70-80%, so I can't lower that either.

**Reply by NickM-27** [reply]
Should be able to set a min size filter on it as I'm sure it's smaller than at person would be

**Reply by Lallassu** [reply]
That sounds like a good idea actually! I will look into that.

**Comment by NickM-27** [comment]
Should be able to set a min size filter on it as I'm sure it's smaller than at person would be
